# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-contain/parsing/contain-computed.html.ini
# Commit: e25543fae7ee
# Full Hash: e25543fae7ee03ac57b9cfbfeb2418c9e2f490fc
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r 4230bdcf07ae -r e25543fae7ee testing/web-platform/meta/css/css-contain/parsing/contain-computed.html.ini
--- a/testing/web-platform/meta/css/css-contain/parsing/contain-computed.html.ini	Wed Jun 22 16:15:57 2022 +0000
+++ b/testing/web-platform/meta/css/css-contain/parsing/contain-computed.html.ini	Wed Jun 22 16:16:59 2022 +0000
@@ -1,8 +1,5 @@
 [contain-computed.html]
-  [Property contain value 'style paint']
-    expected: FAIL
-
-  [Property contain value 'style']
+  [Property contain value 'inline-size layout style paint']
     expected: FAIL
 
   [Property contain value 'style layout paint']
@@ -10,6 +7,3 @@
 
   [Property contain value 'size style layout paint']
     expected: FAIL
-
-  [Property contain value 'inline-size layout style paint']
-    expected: FAIL